renovate.json5

```diff
@@ -20,6 +20,18 @@
       packageNames: ['isolated-vm'],
       allowedVersions: '!/^4\\.4\\.2$/',
     },
+
+    // pnpm 8.6.0 moves to a new lockfile version.
+    {
+      packageNames: ['pnpm'],
+      allowedVersions: '<8.6.0',
+    },
+
+    // see https://github.com/coda/packs-sdk/pull/2642
+    {
+      packageNames: ['open'],
+      allowedVersions: '8.4.2',
```

Comment:
Maybe make this <9.0.0 to allow for minor and patch releases?

Reply:
Also, FYI I forced the off-schedule the creation of the PRs for these so I'd have a place to comment about them

I think closing the renovate PR for `open` will also discourage it from trying to mail PRs for the same major version in the future: https://docs.renovatebot.com/key-concepts/pull-requests/#normal-prs

Reply:
ah that's good to know. I was wondering how to control the renovate PRs.

Reply:
If you haven't seen it, there's also an issue at https://github.com/coda/packs-sdk/issues/2464 where you toggle a checkbox to force off-schedule PR creation or rebase a bunch of PRs at once